VCFv4.3.tex

```diff
@@ -503,41 +503,100 @@ \subsection{VCF tag naming conventions}
 
 \section{INFO keys used for structural variants}
```

Comment:
Agree with push to resolve #133 . Please see my comment [here](https://github.com/samtools/hts-specs/issues/133#issuecomment-322787036).